### PR TITLE
Code Reviewer Role

### PR DESCRIPTION
**Pull request by aneallaryea100**
# CODE REVIEW 
* Play the code reviewer role on my past project
*  Check HTML and CSS best practices 
* Check Linter errors
* Check use of semantic tags in HTML

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous"/>
+    <link rel="stylesheet" href="style.css">
+
+</head>
+<body>
+    <!--NAVIGATION SECTION-->
+<div class="navbar">
+    <div class="logo">
+        <span style="font-size:25px;cursor:pointer">&#9776;</span>
+        <p>BIG-LOGO</p>
```
Comment (aneallaryea100) [inline index.html:17]: Please use the H tag element for your logo for it to stand out.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+    <div class="searchField">
+        <input type="text" name="search" id="searchMe" placeholder="Search">
+        <button class="searhway"><i class="fa fa-search" aria-hidden="true"></i>
+        </button>
+        <button class="voiceway"><i class="fa fa-microphone" aria-hidden="true"></i>
+        </button>
+    </div>
+
+    <div class="icongroup">
```
Comment (aneallaryea100) [inline index.html:28]: Using an unordered list will be great for styling and positioning.
It is preferable in project building

File: index.html [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+        <i class="fa fa-video" aria-hidden="true"></i>
+        <i class="fa fa-th" aria-hidden="true"></i>
```
Comment (aneallaryea100) [inline index.html:30]: Using an unordered list will be great for styling and positioning.
It is preferable in project building

File: index.html [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+                <li>Library</li>
+                <li>History</li>
+                <li>Your videos</li>
+                <li>Watch Later</li>
+                <li>Liked videos</li>
+            </ul>
+        </div>
+          <hr>
+        <div>
+            <p>SUBSCRIPTIONs</p>
```
Comment (aneallaryea100) [inline index.html:65]: H tag will be great and easy to recognize than the p tag
It more preferable in project building

File: index.html [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+        <button class="voiceway"><i class="fa fa-microphone" aria-hidden="true"></i>
+        </button>
+    </div>
+
+    <div class="icongroup">
+        <i class="fa fa-video" aria-hidden="true"></i>
+        <i class="fa fa-th" aria-hidden="true"></i>
+        <i class="fa fa-bell" aria-hidden="true"></i>
+        <i class="fa fa-user-circle" aria-hidden="true"></i>
+    </div>
```
Comment (aneallaryea100) [inline index.html:33]: Please use an unordered list to list your icons 
It makes styling and positioning easier in your project

File: style.css [hunk 6]
```diff
@@ -0,0 +1,167 @@
+*{
+    margin:0;
+    padding: 0;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
```
Comment (aneallaryea100) [inline style.css:4]: Please put the font family in your the body

Comment (aneallaryea100) [reply style.css:4]: When the font family is in the body tag is applies to all element in your HTML body which is a best practice.

File: index.html (hunk 1 above)
Comment (aneallaryea100) [reply index.html:17]: Kindly add an A tag within your H tag to make your logo clickable and is part of best practices

**Review by Ismailco — APPROVED** [review]
### Hi @aneallaryea100,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

-  Kindly make sure to pay attention to the grammar in your comments

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by aneallaryea100** [comment]
Ready for merge.

**Pull request merged**